core/server/common/src/main/java/alluxio/master/journal/ufs/UfsJournalLogWriter.java

```diff
@@ -229,4 +392,28 @@ public synchronized void close() throws IOException {
     closer.close();
     mClosed = true;
   }
+
+  /**
+   * @return number of journal entries to flush stored in the queue
+   */
+  @VisibleForTesting
+  synchronized  int getNumberOfJournalEntriesToFlush() {
+    return mEntriesToFlush.size();
+  }
+
+  @VisibleForTesting
+  synchronized JournalOutputStream getJournalOutputStream() {
+    return mJournalOutputStream;
+  }
+
+  /**
+   * @return the current underlying {@link DataOutputStream} to write journal entries
+   */
+  @VisibleForTesting
+  synchronized DataOutputStream getUnderlyingDataOutputStream() {
```

Comment:
we only use this method for testing that whitebox works properly - we should get rid of it and save ourselves a `@VisibleForTesting` annotation.